unpack-build-maven-plugin/src/main/java/org/kie/unpackbuildplugin/util/PomManipulationUtils.java

```diff
@@ -0,0 +1,57 @@
+package org.kie.unpackbuildplugin.util;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.function.Consumer;
+
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.apache.maven.model.io.xpp3.MavenXpp3Writer;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+
+public class PomManipulationUtils {
```

Comment:
Minor thing, but please add an empty private constructor as it shouldn't be possible to instantiate util classes.

Reply:
Done.